static/src/js/App.jsx

```diff
@@ -249,7 +250,7 @@ export const App = () => {
     },
     {
       path: '/404',
-      element: <Page header={<div />} title="404 Not Found" pageType="secondary">Not Found :(</Page>
+      element: <ErrorPageNotFound />
```

Comment:
Which makes sense in the old rails because that's what the page has when it's rendering properly. 
<img width="1535" alt="Screenshot 2024-07-17 at 9 32 41 AM" src="https://github.com/user-attachments/assets/76c12844-3bb7-4c80-a0a6-f130beed02fc">
But our new rails application doesn't have the blue header bar. Should I render a blank side column (with NASA logo and MMT header) instead for consistency? 
![Screenshot 2024-07-17 at 9 34 03 AM](https://github.com/user-attachments/assets/0e801f6e-3efc-443f-adaf-1a6d7f6814ff)

Reply:
We do still have the blue header. It is whats used when a user is not authenticated. I am suggesting you use that one. We would just need to add a prop that removes the login button.

Reply:
If my fix isn't satisfactory, could you point me towards the file where the blue bar already exists?

Reply:
It is currently used in `static/src/js/components/LayoutUnauthenticated/LayoutUnauthenticated.jsx`